tests/tensorflow2/test_keras.py

```diff
@@ -37,7 +37,7 @@ def helper_keras_fit(
     eager=True,
     steps=None,
     add_callbacks=None,
-    run_eagerly=None,
+    run_eagerly=False,
```

Comment:
is this meant for model.compile()? where is it being used?

Reply:
yes